lib/ReactViews/Custom/Chart/BottomDockChart.jsx

```diff
@@ -402,6 +396,12 @@ class Plot extends React.Component {
 }
 
 class XAxis extends React.PureComponent {
```

Comment:
Accepting only simple types as props so that `shouldComponentUpdate` of the PureComponent works as expected.